cpp/conda.recipe/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+set -x
+
+cd $RECIPE_DIR
+
+# Build dependencies
+export FLATBUFFERS_HOME=$PREFIX
+export PARQUET_HOME=$PREFIX
+
+cd ..
+
+rm -rf conda-build
+mkdir conda-build
+
+cp -r thirdparty conda-build/
+
+cd conda-build
+pwd
+
+# Build googletest for running unit tests
+./thirdparty/download_thirdparty.sh
+./thirdparty/build_thirdparty.sh gtest
+
+source thirdparty/versions.sh
+export GTEST_HOME=`pwd`/thirdparty/$GTEST_BASEDIR
+
+if [ `uname` == Linux ]; then
+    SHARED_LINKER_FLAGS='-static-libstdc++'
+elif [ `uname` == Darwin ]; then
+    SHARED_LINKER_FLAGS=''
+fi
+
+cmake \
+    -DCMAKE_BUILD_TYPE=debug \
```

Comment:
Is `debug` just used here until we have more stable dev versions? Or do you plan to do debug dev version? 
I'd prefer if we would make a ReleaseWithDebugSymbols build_type for dev versions.